<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WGConfigWidget</class>
 <widget class="QWidget" name="WGConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSelector">
      <attribute name="title">
       <string>Color Selector</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Hue Ring Appearance:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbSelectorRenderingMode">
         <item>
          <property name="text">
           <string>Static</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dynamic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dynamic + Static Edge</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="KisPopupButton" name="btnSelectorShape">
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbColorModel">
         <item>
          <property name="text">
           <string>HSV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HSL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HSI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HSY'</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="grpQuickSettingsMenu">
         <property name="title">
          <string>Show &amp;Quick Settings Menu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Favorite Selector Layouts:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KisPopupButton" name="btnFavoriteSelectors">
            <property name="text">
             <string>Select...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Selection Color Space:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cmbSelectionColorSpace">
           <item>
            <property name="text">
             <string>Layer/Mask</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Image</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisPopupButton</class>
   <extends>QPushButton</extends>
   <header>KisPopupButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
